### DISCUSSION TITLE
How can I schedule tweets natively?

### DISCUSSION THREAD

**Discussion by AustinGitHub** [tweepy/tweepy#1782] [Questions] [answered]
Can we get support for this (Been nearly a year since it was asked about) https://developer.twitter.com/en/docs/twitter-ads-api/creatives/api-reference/scheduled-tweets#post-accounts-account-id-scheduled-tweets

They have it available and I think it would be very helpful to schedule the tweets automatically through python. 

### Discussed in https://github.com/tweepy/tweepy/discussions/1525

<div type='discussions-op-text'>

<sup>Originally posted by **sevenleo** February 13, 2021</sup>
Can I use Tweepy API to schedule posts?

![image](https://user-images.githubusercontent.com/5843634/107843592-3ae73c00-6dab-11eb-8ea9-c131cdf4763b.png)
</div>

**Comment by Harmon758** [comment, marked as answer]
The Twitter Ads API is for advertisers, not normal user accounts, and there are no plans to support it.
You should look into using the [official Twitter supported and maintained Ads API SDK for Python](https://github.com/twitterdev/twitter-python-ads-sdk) instead.
